source/partials/release_notes/_release-20-2-0.md.erb

```diff
@@ -0,0 +1,84 @@
+<h4>Support for Rules on Config Repositories (Pipeline as Code)</h4>
+
+In order to provide fine-grained access control over the entities that are created by a config repository, GoCD adds support for defining rules on config repositories that specifies which environments, pipeline groups, pipelines the repository can refer to.
+Please refer to the [Specifying rules on Config Repository](https://docs.gocd.org/20.2.0/advanced_usage/pipelines_as_code.html#specifying-rules) for more information.
+
+
+<h4>Removal of Default SSL Support from GoCD</h4>
+
+Starting with GoCD 20.2.0, GoCD will no longer support configuring TLS. See [this Github Issue](https://github.com/gocd/gocd/pull/7669#issuecomment-580290432) for more information.
+
+
+<h4>Other Improvements</h4>
+
+* Starting this release, Alpine 3.11 and CentOS 8 based docker images for GoCD Server are <%= link_to 'available', 'https://hub.docker.com/search?q=gocd%2Fgocd-server&type=image' %>.
+* <%= link_to_issue 6031, 'Improve Pipeline Material auto update mismatch error' %>
+* <%= link_to_issue 7736, 'Improvements to wrapper properties examples' %>
+* <%= link_to_issue 7748, 'Improve Logging - Add plugin load/update/remove logs' %>
+* <%= link_to_issue 7713, 'Provide additional information about GoCD deprecated APIs' %>
+* <%= link_to_issue 3194, 'Add the possibility to retrieve template parameters through the template API' %>
+
+
+<h4>Bug fixes</h4>
+
+* <%= link_to_issue 7737, 'Fixed issue with moving pipeline between groups' %>
+* <%= link_to_issue 7676, 'Fixed invalid run_instance_count during pipeline export' %>
+* <%= link_to_issue 7822, 'Missing response body error when submitted Pipeline JSON has duplicate job name error' %>
+* <%= link_to_issue 7824, 'Pipeline Config API does not return any error message for invalid job resources' %>
+* <%= link_to_issue 7830, 'Don\'t showcase pending agents in edit agents modal' %>
+* <%= link_to_issue 7853, 'Cannot search when type config repo name in the search box' %>
```

Comment:
Can be reworded as `Improve search on Config Repo SPA` ??

Reply:
I generally prefer keeping issue title same as of the Github issue title unless its too off the topic. 
It helps users (and us as well) to identify the issue immediately.

But yeah, we can make this change.

Reply:
I share your views but in this case it is better to be clear from a view point of what bug was fixed.